pa_config/rca.conf

```diff
@@ -36,5 +36,7 @@
     // How often the sqlite file be repeated in seconds. This file contains RCAs and therefore rotating it too frequently
     // might not be as fruitful as there might not be any data.
     "rotation-period-seconds": 21600
-  }
+  },
+
+  "muted-rcas": "rca1, rca2"
```

Comment:
We should remove this, right ?

Reply:
Right. Have replaced this with an empty string.